data_uploader/netbox_api/netbox_ipam.py

```diff
@@ -0,0 +1,52 @@
+from data_uploader.netbox_api.netbox_dcim import NetboxDcim
+from data_uploader.netbox_api.netbox_connection import NetboxApi
+import pynetbox
+
+class NetboxIpam:
+    def __init__(self, url, token, cert):
+        self.url = url
+        self.token = token
+        self. cert = cert
+        self._netbox_api = NetboxApi.api_object(url, token, cert)
```

Comment:
If this is going to be identical in every `Netbox*` class I would extract it to a netbox base as follows:

```python
class NetboxBase:
  def __init__(self, url, token, cert):
        self.url = url
        self.token = token
        self. cert = cert
        self._netbox_api = NetboxApi.api_object(url, token, cert)
```

```python
class NetboxBase(NetboxBase):
  def __init__(self, url, token, cert):
     super().__init__(url, token, cert)
```